### PR TITLE
use full qualified function calls in Property/Text->serialize()

### PR DESCRIPTION
**Pull request by staabm**
explcitly use full qualified function calls so PHP can generate faster opcodes and doesn't need to check the local namespace for equal named functions before using the native function.

applied the changes only in the HOT path.

### CODE REVIEW

**Comment by staabm** [comment]
@gharlan could you double check whether this change improves performance for you?

File: lib/Property.php [hunk 1]
```diff
@@ -247,11 +247,11 @@ function serialize() {
         $str .= ':' . $this->getRawMimeDirValue();
 
         $out = '';
-        while (($len = strlen($str)) > 0) {
+        while (($len = \strlen($str)) > 0) {
             if ($len > 75) {
```
Comment (staabm) [inline lib/Property.php]: @evert where does this 75byte limitation come from? is this defined in the spec?

Comment (gharlan) [reply lib/Property.php]: @staabm for vcard you can find it here: https://tools.ietf.org/html/rfc6350#section-3.2

**Comment by staabm** [comment]
with this changes php seems to be able to eliminate all the `strlen()` calls (I guess via opcache optimizations). In our case this means ~420.000 less function invocations.

**Comment by staabm** [comment]
including the last commit 3086b8f we get another 2 seconds back.

grafik

**Pull request merged**